critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -46,6 +46,16 @@
           <br /><small class="text-warning"><em>{{ _('You cannot change the license after the review is published.') }}</em></small>
         </div>
       </div>
+      {% if not current_user.license_choice %}
+        <div class="form-group">
+          <div class="col-sm-12">
+            <label class="col-sm-4"> {{ _('Remember this license choice for further preference') }} </label>
+            <div class="checkbox">
```

Comment:
It would be good to put the checkbox before the label. 
![1](https://user-images.githubusercontent.com/12391372/38299473-cf53f5d0-3817-11e8-8ec0-185e87ad61f7.png)

It would be good if all the checkboxes and radio buttons are on the left side.